apps/user-portal/src/components/account-recovery/options/email-recovery.tsx

```diff
@@ -37,77 +38,62 @@ interface EmailRecoveryProps {
  */
-export const EmailRecovery: React.FunctionComponent<EmailRecoveryProps> = (
-    props: EmailRecoveryProps
-): JSX.Element => {
+export const EmailRecovery: React.FunctionComponent<EmailRecoveryProps> = (props: EmailRecoveryProps): JSX.Element => {
     const [email, setEmail] = useState("");
     const [editedEmail, setEditedEmail] = useState("");
     const [isEdit, setIsEdit] = useState(false);
     const { t } = useTranslation();
     const { onNotificationFired } = props;
 
-    const handleUpdate = () => {
+    const handleUpdate = (emailParam: string) => {
         const data = {
             Operations: [
                 {
                     op: "replace",
                     value: {}
                 }
             ],
-            schemas: [
-                "urn:ietf:params:scim:api:messages:2.0:PatchOp"
-            ]
+            schemas: ["urn:ietf:params:scim:api:messages:2.0:PatchOp"]
         };
 
         data.Operations[0].value = {
-            emails: [editedEmail]
+            emails: [emailParam]
         };
 
-        updateProfileInfo(data)
-            .then((response) => {
-                if (response.status === 200) {
-                    onNotificationFired({
-                        description: t(
-                            "views:components.accountRecovery.emailRecovery.updateEmail.notifications.success.description"
-                        ),
-                        message: t(
-                            "views:components.accountRecovery.emailRecovery.updateEmail.notifications.success.message"
-                        ),
-                        otherProps: {
-                            positive: true
-                        },
-                        visible: true
-                    });
-                    getProfileInfo()
-                        .then((res) => {
-                            setEmailAddress(res);
-                        });
-                    setIsEdit(false);
-                } else {
-                    onNotificationFired({
-                        description: t(
-                            "views:components.accountRecovery.emailRecovery.updateEmail.notifications.error.description"
-                        ),
-                        message: t(
-                            "views:components.accountRecovery.emailRecovery.updateEmail.notifications.error.message"
-                        ),
-                        otherProps: {
-                            negative: true
-                        },
-                        visible: true
-                    });
-                }
-            });
-    };
-
-    /**
-     * The following function handles the change of state of the input fields.
-     * The id of the event target will be used to set the state.
-     * @param event
-     */
-    const handleFieldChange = (event) => {
-        setEditedEmail(event.target.value);
-        event.preventDefault();
+        updateProfileInfo(data).then((response) => {
```

Comment:
This only handles `then` and will not show notifications for error codes like 400 etc.